meta-lmp-base/recipes-support/lmp-el2go-auto-register/lmp-el2go-auto-register.bb

```diff
@@ -4,7 +4,10 @@ SECTION = "devel"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
 
-SRC_URI = "git://github.com/foundriesio/lmp-el2go-auto-register.git;protocol=https;branch=main"
+SRC_URI = "git://github.com/foundriesio/lmp-el2go-auto-register.git;protocol=https;branch=main \
+	file://default.env \
```

Comment:
Will this fail the build if the files is not present in meta-sub layer?

Reply:
No, it will not fail when the files not present in meta-sub layer. In this case bitbake will pick up the default empty files added on this commit.

Reply:
ah, I didn't notice empty files. I guess this is OK.